assets/dev/scss/editor/_control-scrubbing.scss

```diff
@@ -0,0 +1,26 @@
+.e-scrubbing {
+	
+	&, * {
+		cursor: e-resize;
+	}
+
+	input.e-scrubbing--active {
```

Comment:
Please compare the performance
```suggestion
	.e-scrubbing--active:is( input ) {
```

Reply:
Please let's talk about it.

Reply:
Seems like input.class is faster then class:is(input)
each time I checked - the first one won.

![image](https://user-images.githubusercontent.com/101200834/167283560-c2129482-1a52-4a6f-9ca7-ca8b53f1de52.png)

![image](https://user-images.githubusercontent.com/101200834/167283582-c6e60d8d-d5fa-4c3f-8762-55c79db26fc2.png)